components/ConfigureQuiz/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import Select from "react-select";
+
+import Button from '../Button';
+
+import { DIFFICULTY_OPTIONS } from './constants';
+
+type Props = {
+    onDifficultySelect: Function,
```

Comment:
This is a typescript comment, so I'm not sure if you should do anything about it, but a "Function" type allows any function to be passed, whereas I would recommend a specific function signature to be provided so that its interface is explicit in the application. Would you be able to update that?

Reply:
What would the structure of a specific function signature look like? I will certainly try.